src/main/java/uk/gov/companieshouse/filingmock/processor/strategy/AcceptanceStrategyFactory.java

```diff
@@ -25,8 +25,8 @@ public static AcceptanceStrategy getStrategy(Transaction submission) {
             return REA;
         } else if (submissionType.contains("insolvency")) /* There are multiple insolvency types e.g. insolvency#600 but all will start with "insolvency," and should use the same strategy */ {
             return INSOLVENCY;
-        } else if (submissionType.contains("cessation")) {
-            return CESSATION;
+        } else if ("psc-filing#cessation".equals(submissionType)) {
```

Comment:
the submissionType in the chips filing mock will be "psc-filing#cessation#individual" for an individual psc that is doing a PSC07 which makes me think when future functionality is added it could also be  "psc-filing#cessation#corporate-entity" or "psc-filing#cessation#legal-person" depending on which type of PSC it is.

Perhaps this line needs to slightly change to take this into account.

Reply:
Changed to `startsWith` instead. Thanks!